docs/content/latest/explore/ysql-language-features/gin.md

```diff
@@ -262,17 +260,17 @@ If write performance and storage aren't an issue, it may be worth creating a GIN
 
 ### Presplitting
 
-By default, `ybgin` indexes use a single range-partitioned tablet.
-Like regular tables and indexes, it is possible to presplit a `ybgin` index to multiple tablets at specified split points.
+By default, `gin` indexes use a single range-partitioned tablet.
+Like regular tables and indexes, it is possible to presplit a `gin` index to multiple tablets at specified split points.
```

Comment:
```suggestion
By default, GIN indexes use a single range-partitioned tablet.
Like regular tables and indexes, it is possible to presplit a GIN index to multiple tablets at specified split points.
```

This is incorrect when you look at "`gin`" as the access method name.  `gin` is for temporary tables, so it doesn't even use tablets.  It may be confusing to bring up the technically correct "`ybgin`" here, so I suggest saying the intentionally ambiguous "GIN".